torchvision/models/googlenet.py

```diff
@@ -112,14 +119,15 @@ def _initialize_weights(self):
                 nn.init.constant_(m.weight, 1)
                 nn.init.constant_(m.bias, 0)
 
-    def forward(self, x):
-        # type: (Tensor) -> GoogLeNetOutputs
+    def _transform_input(self, x):
```

Comment:
We need type annotations or else torchscript will not work for those models, can you add some basic type annotation? In this case, `Tensor -> Tensor`, which I think is the default for torchscript, so maybe in this case it might not be needed, but it's better to have it than not to have it.